dbt_project.yml

```diff
@@ -2,7 +2,7 @@ config-version: 2
 name: 'quickbooks'
  
 version: '0.6.0'
-require-dbt-version: [">=1.3.0", "<2.0.0"]
+require-dbt-version: [">=1.0.0", "<2.0.0"]
```

Comment:
this should be 1.3 for post migration merges

Reply:
Good catch, updating!